cmd/verrazzano-operator/main.go

```diff
@@ -49,25 +37,26 @@ func main() {
 
 	err := prepare()
 	if err != nil {
-		zap.S().Fatalf("Error loading manifest: %s", err.Error())
+		zap.S().Fatalf("Failed loading manifest: %v")
```

Comment:
Should err still be in the argument list?

Reply:
fixed